java/dataset/src/test/java/org/apache/arrow/dataset/file/TestFileSystemDataset.java

```diff
@@ -126,6 +128,7 @@ public void testParquetBatchSize() throws Exception {
     checkParquetReadResult(schema, writeSupport.getWrittenRecords(), datum);
 
     AutoCloseables.close(datum);
+    AutoCloseables.close(factory);
```

Comment:
combine this line with the previous?  close can take multiple parameters i believe.

Reply:
They are not able to be combined directly. The first line calls code https://github.com/apache/arrow/blob/0a3cbd0b7337b464ef55619672727ebe6f65ad05/java/memory/memory-core/src/main/java/org/apache/arrow/util/AutoCloseables.java#L82-L109
second line calls
https://github.com/apache/arrow/blob/0a3cbd0b7337b464ef55619672727ebe6f65ad05/java/memory/memory-core/src/main/java/org/apache/arrow/util/AutoCloseables.java#L74-L76